src/main.rs

```diff
@@ -98,7 +99,11 @@ pub fn start(mut os_input: Box<dyn OsApi>, opts: Opt) {
     let command_is_executing = CommandIsExecuting::new();
 
     delete_log_dir().unwrap();
-    delete_log_file().unwrap();
+
+    if opts.clear_log_file {
+        // TODO: Remove this once it is decided that it goes into the logs dir
```

Comment:
@henil Was it you who wanted to move the logging files all to one directory?

Reply:
Yes, but it was decided it should be done in different PR

Reply:
Yep, just wanted to make sure that you noticed this

Reply:
This has been decided and I am opening a PR for it. So I think this check and `clear_log_file` from the `Opt` struct should be removed.